### PR TITLE
Integrate two addons

### PR DESCRIPTION
**Pull request by FinalAngel**
this PR integrates 

- aldryn-newsblog
- aldryn-style

### CODE REVIEW

**Comment by intellectronica** [comment]
:+1:

File: src/settings.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
-gettext = lambda s: s
+_ = lambda s: s
```
Comment (intellectronica) [inline src/settings.py]: eh?

Comment (czpython) [inline src/settings.py]: This is no longer needed, please use ```from django.utils.translations import ugettext_lazy as _```

File: src/settings.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+from django.utils.translations import ugettext_lazy as _
```
Comment (FinalAngel) [inline src/settings.py]: this fails the test while of course ``_ = lambda s: s`` did not - what am I mussing @intellectronica

**Pull request merged**